Platon/backend/app/auth_system/views.py

```diff
@@ -191,16 +190,18 @@ def get(requester_id, self):
                     if Follow.query.filter_by(follower_id=requester_id, following_id=form.user_id.data).first() is not None:
                         following_status = 1 # Represents that the requester user follows the requested user.
                     elif FollowRequests.query.filter_by(follower_id=requester_id, following_id=form.user_id.data).first() is not None:
-                        following_status = 0 # Represents that the requester user has already sent a follow request to the requested user.
+                        following_status = 0 # Represents that the requester user has already sent a follow request to the requested user and it is pending.
                     else:
                         following_status = -1 # Represents that the requester user does not follow the requested user and has not yet sent a request to follow them.
                     
```

Comment:
@oykuyilmaz1 @darktheorys @ahmetdadak 

After this pull request is merged and deployed to our server, you will be able to get whether the requester user follows the requested user or not when you send a "Get User" request.

For example:
Requester user: User 1
Requested user: User 2

If User 1 already follows User 2, "following_status" field will have the value "1".
If User 1 has sent a follow request to User 2; but User 2 has not yet responded to that request, "following_status" field will have the value "0".
If User 1 is not currently following User 2 and does not have a pending follow request, "following_status" field will have the value "-1".